src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,22 @@
+
+
+import FancyBox from "./FancyBox";
+
+export default function ArticleCard({ article }) {
+    return (
```

Comment:
There should always be `<li></li>` with in the `<ul>`or `<ol>`, so better to wrap everything in `<li>`tag here